guide/inheritance-wallet/backup.md

```diff
@@ -0,0 +1,279 @@
+---
+layout: guide
+title: Wallet backup
+description: This page gives an overview of how the Joneses backup their savings wallet.
+nav_order: 4
+parent: Inheritance wallet
+permalink: /guide/inheritance-wallet/backup/
+main_classes: -no-top-padding
+image_base: /assets/images/guide/inheritance-wallet/
+images_wallet-backup:
+    - file: wallet-backup/app-home-backup-before
+      alt: Image of the app home screen showing a reminder to backup the wallet.
+      caption: The app shows a reminder to save the backup kit.
+    - file: wallet-backup/wallet-settings-backup-kit
+      alt: Screen showing instructions on how to store the wallet backup.
+      caption: The backup page highlights the most important points about the wallet backup and provides a link to learn more.
+    - file: wallet-backup/backup-save-dialog
+      alt: Placeholder screen representing the operating system native file download dialog.
+      caption: The file is saved using the native file download flow.
+    - file: wallet-backup/backup-success
+      alt: Screen showing a success message.
+      caption: The success screen reminds the user once again to not store the backup kit in the same place as the private key backups.
+    - file: wallet-backup/app-home-backup-done
+      alt: Image of the home screen that does not show the backup reminder anymore.
+      caption: The app home screen does not show the backup reminder anymore.
+---
+
+<!--
+
+Editor's notes
+
+This page covers how users backup their wallet and an example approach of how to store the backup material.  
+
+Illustration sources
+
+https://www.figma.com/file/h5GP5v5dYfpXXfEUXf6nvC/Inheritance-wallet?type=design&node-id=6293%3A21917&mode=design&t=I2e3qgqYRGpAGyaQ-1 
+
+-->
+
+# Wallet backup 
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+
+With the family savings wallet in place, establishing a robust backup strategy is crucial. However, there's no universal solution for wallet backups. Each family must tailor their approach based on their unique circumstances, considering factors such as:
+
+- Relationships between family members
+- Amount of funds stored
+- Access to secure storage locations
+- Technical comfort level of family members
+- Desired balance between security and accessibility
+
+The backup scheme outlined below is one potential approach. It aims to balance security, privacy, and ease of use, but may not suit every family's needs. As you read through this guide, consider how you might adapt these strategies to best fit your family's situation.
+
+## What needs to be backed up?
+There are two parts to backing up a multi-key wallet: the private keys as well as the wallet configuration.
+
+- **Private keys**: Secure backups of all private keys used in the wallet.
+- **Wallet descriptor**: A small piece of text containing the wallet configuration.
```

Comment:
Would suggest to use one term or wallet descriptor or wallet configuration since elsewhere it mentions that these can be used interchangeably.